ownCloud/Messages/MessageQueueExample.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  MessageQueueExample.swift
+//  ownCloud
+//
+//  Created by Felix Schwarz on 14.06.20.
+//  Copyright © 2020 ownCloud GmbH. All rights reserved.
+//
+
+/*
+ * Copyright (C) 2020, ownCloud GmbH.
+ *
+ * This code is covered by the GNU Public License Version 3.
+ *
+ * For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+ * You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+ *
+ */
+
+import UIKit
+import ownCloudSDK
+
+/*
+		// Example of using this example:
```

Comment:
Maybe better: `// Example of using the MessageQueueExample `

Reply:
Good idea. Changed that.